### DISCUSSION TITLE
PDF Generated Meeting Guide

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1508] [Q&A]
Hi Y'all, 

When you try to print the meeting guide by just the day we are getting a 500 server error. 
![image](https://github.com/user-attachments/assets/d3bb241b-cbbb-4bd8-aac6-e456db3bcfd2)

**Comment by gkovats** [comment]
What meeting list are you trying to print? There's an upper limit to how large a list of meetings it'll print.
